### ISSUE TITLE
Private chain needs transaction fees

### ISSUE THREAD

**Issue by hacktar** [inzhoop-co/LETH#62]
as reported from users:
"For a private blockchain, I do not really need an ETH as a transaction fee. Unfortunately, LETH is set up by default to require ETH for transaction. Unfortunately, if a user does not own ETH, but only wants to send the custom token, this will not work."

**Comment by hacktar** [comment]
The transaction fees (aka [gas](https://ethereum.stackexchange.com/questions/3/what-is-meant-by-the-term-gas)) are needed even in a private chain because they are part of the blockchain's functioning mechanism. 
The difference in a private chain is that ETH could be mined easily and they don't have a value in FIAT.

see further [details](https://blockgeeks.com/guides/ethereum-gas-step-by-step-guide/)

**Comment by ed-prinz** [comment]
Yes - Thank you! Mayby it is possible to hide the transaction fee scroll bar even LETH is connected with private chain? Some Users do not understand why they need this to pay some food with digital food stamps...the transaction fee might have a default value that can not be changed. Of course only if it is easy to adjust, with a condition (if private chain then hide transaction fee bar) otherwise learning by doing is also a good way for the users ;)
